src/signer/signer.py

```diff
@@ -257,50 +227,62 @@ def run_signer(wallet_id=None, key_pair_seed=None, nonce_seed=None):
-                quorum, relay_manager, private_key)
+                quorum, relay_manager, nostr_private_key)
+            # TODO: nonce_seed is currently required
             wallet = Wallet(wallet_id, key_pair_seed=key_pair_seed, nonce_seed=nonce_seed)
-        elif user_input.lower() == "send pk":
+
+        elif user_input.lower() == SignerCommands.SEND_PUBLIC_KEY.value:
             logging.info("Generating and posting the public key...")
-            handle_create_xpub(wallet, relay_manager, private_key)
-        elif user_input.lower() == "address":
+            handle_create_xpub(wallet, relay_manager, nostr_private_key)
+
+        elif user_input.lower() == SignerCommands.GENERATE_ADDRESS.value:
             # TODO bug: you cannot sign or spend with out getting an address first
             logging.info("Generating a new address...")
             address_payload = handle_get_address(
-                wallet, 0, relay_manager, private_key)
+                wallet, 0, relay_manager, nostr_private_key)
 
             wallet.set_cmap(address_payload['cmap'])
             wallet.set_pubkey_agg(address_payload['pubkey_agg'])
             logging.info(f"Got address {address_payload['address']}")
 
-        elif user_input.lower() == "spend":
+        elif user_input.lower() == SignerCommands.SPEND.value:
             logging.info("Preparing to spend funds...")
             txid = input("Enter previous tx id: ")
             index = int(input("Enter output index: "))
-            new_address = input("Where would you like to send sats to: ")
-            sats = int(input("How much are we spending: "))
+            new_address = input("Destination address (where would you like to send funds to?): ")
+            sats = int(input("Amount in satoshis (how much are we spending?): "))
 
             spend_request_id = handle_spend(
-                [txid, index], new_address, sats, wallet, relay_manager, private_key)
+                [txid, index], new_address, sats, wallet, relay_manager, nostr_private_key)
             wallet.set_current_spend_request_id(spend_request_id)
             logging.info(
                 f'Your spend request id {spend_request_id}, next provide nonces and signatures!!')
-        elif user_input.lower() == "sign":
+
+        elif user_input.lower() == SignerCommands.SIGN.value:
             spend_request_id = input("Provide a spend request id: ")
             spend_request_id = handle_sign_tx(
-                spend_request_id, wallet, relay_manager, private_key)
+                spend_request_id, wallet, relay_manager, nostr_private_key)
+
         else:
-            logging.info("Invalid command. Please enter one of the following: 'new wallet', 'send pk', ' address', 'spend', 'sign'")
\ No newline at end of file
+            possible_commands = [c.value for c in SignerCommands]
+            logging.info("Invalid command. Please enter one of the following: %s", possible_commands)
```

Comment:
```suggestion
            logging.info("Invalid command. Please enter one of the following: %s", possible_commands)
            
```

Adding new line to end of file